thali/thalireplicationmanager.js

```diff
@@ -112,6 +148,18 @@ ThaliReplicationManager.prototype.stop = function () {
   });
 };
 
+function informDeviceIdentityListeners(err, publicKeyHash) {
```

Comment:
Shouldn't this code be hanging out with the rest of the identity set code? It's odd to separate them. But I realize this was probably a consequence of my comment about moving the require not being clear enough. Sorry. But could you please re-unite informDeviceIdentityListeners and clearDeviceIdentityListeners with getDeviceIdentity?